srcpkgs/python3-pytest-flake8/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-pytest-flake8'
+pkgname=python3-pytest-flake8
+version=1.0.7
+revision=1
+wrksrc=pytest-flake8-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-pytest flake8"
+checkdepends="$depends"
+short_desc="Pytest plugin to check FLAKE8 requirements"
+maintainer="Paper <paper@tilde.institute>"
+license="BSD-3-Clause"
+homepage="https://pypi.org/project/pytest-flake8/"
+changelog="https://raw.githubusercontent.com/tholo/pytest-flake8/master/CHANGELOG"
+distfiles="${PYPI_SITE}/p/pytest-flake8/pytest-flake8-${version}.tar.gz"
+checksum=f0259761a903563f33d6f099914afef339c085085e643bee8343eb323b32dd6b
+
+do_check() {
+	PYTHONPATH=. pytest
+}
```

Comment:
You don't need this. The build style will automatically call `python3 -m pytest` when `python3-pytest` is available, which implicitly adds `.` to `PYTHONPATH`.